crypto-common/src/lib.rs

```diff
@@ -154,7 +154,7 @@ pub trait KeyInit: KeySizeUser + Sized {
         if key.len() != Self::KeySize::to_usize() {
             Err(InvalidLength)
         } else {
-            Ok(Self::new(Key::<Self>::from_slice(key)))
+            Ok(Self::new(Key::<Self>::ref_from_slice(key)))
```

Comment:
Note: this method exists to simplify migrating `generic-array`-based code but I'd like to at least deprecate it before the next release.

The code here should be replaced with `TryFrom` which I'll do before removing draft.